jibri/rootfs/etc/supervisor/conf.d/20_xorg.conf

```diff
@@ -0,0 +1,6 @@
+[program:xorg]
+command=/bin/bash -c "/usr/bin/service_wrapper_xorg"
```

Comment:
Why is xorg necessary ?

Reply:
jibri is actually launching some kind of chromium and screencapturing it ^^

Reply:
There is way worse: it needs alsa kernel module for the audio capture. There are ways to use pulseaudio instead so we could run it on EKS for example. I have about 10 bookmarks avail on request to do so if you want to have a look @Arkaniad , I was planning to do so one day when I have time ^^

Reply:
As of right now it is working on EKS so whatever magic it does with ALSA seems to work fine